h5py/h5d.pyx

```diff
@@ -430,3 +431,65 @@ cdef class DatasetID(ObjectID):
                 efree(offset)
                 if space_id:
                     H5Sclose(space_id)
+
+    IF HDF5_VERSION >= (1, 10, 2):
+
+        def read_direct_chunk(self, offsets, filter_mask=0xFFFF, PropID dxpl=None):
+            """ (offsets, H5Z_filter_t filter_mask=[], PropID dxpl=None)
```

Comment:
I'm pretty sure the default value `[]` and the type `H5Z_filter_t` are both wrong. The [hdf5 docs](https://confluence.hdfgroup.org/display/HDF5/H5D_READ_CHUNK) say the type is uint32_t.

H5Z_filter_t is a numeric ID for a single filter. This value is a mask to select filters in a pipeline.

Reply:
Fixed